gulp-file.ts

```diff
@@ -87,7 +95,11 @@ gulp.task('build:src', ['lint', 'clean'], () => {
     return build(files.src.ts, paths.src.js);
 });
 
-gulp.task('build', ['build:tests', 'build:src']);
+gulp.task('build:appsrc', ['lint', 'clean'], () => {
+    return build(files.site.ts, paths.site.app);
+});
+
```

Comment:
I'm not sure if this is where we want to be building all of our appsrc files. In my mind the ./app directory would contain the HTML, techstudio.js(.min, .map) and the app.js(.min, .map) and that's it. The intermediate appsrc .js files might belong elsewhere? But I don't know where..